### PR TITLE
Create CONTENT_TEMPLATE.md

### PR DESCRIPTION
**Pull request by deweydell**
We don't have an issue template for adding new content to the website without making any code changes. For example for issues like this one: https://github.com/WriteSpeakCode/writespeakcode.com/issues/33

### Proposed changes

- Add github pull request template for adding content 

### Requested feedback
- Is there anything confusing about the template?
- Any suggestions for updating the copy of the template so people know how to use it?

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/CONTENT_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+name: Add content to the website
+about: For adding content - can be completed entirely in CMS
```
Comment (coilysiren) [inline .github/ISSUE_TEMPLATE/CONTENT_TEMPLATE.md]: non-blocking suggestion

```suggestion
about: For adding content - can be completed entirely in CMS, or via a github pull request
```

You can update content with a pull request, yea?

File: .github/ISSUE_TEMPLATE/CONTENT_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+---
+
+### Content type
+<!--
+Is this an issue to add or update a speaker? sponsor? copy? metadata? etc
+-->
+
+
+#### Content Details
+<!-- Add all the information for the content and upload any images here if applicable -->
```
Comment (coilysiren) [inline .github/ISSUE_TEMPLATE/CONTENT_TEMPLATE.md]: non-blocking suggestion

```suggestion
<!-- Add information about the content and upload any images here, if applicable -->
```

**Review by coilysiren — DISMISSED** [review]
I love it 💖

File: .github/ISSUE_TEMPLATE/CONTENT_TEMPLATE.md (hunk 1 above)
Comment (deweydell) [reply .github/ISSUE_TEMPLATE/CONTENT_TEMPLATE.md]: yes ✨

**Pull request merged**